.github/workflows/periodic-audit.yaml

```diff
@@ -0,0 +1,12 @@
+name: Cargo security audit
+on:
+  schedule:
+    - cron: '0 0 * * 0'
+jobs:
+  audit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: rustsec/audit-check@v1.4.1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
can we just have github dependabot do this? I'd imagine it supports this

Reply:
will address in a future PR